$NetBSD: patch-aa,v 1.3 2013/01/22 10:18:42 mef Exp $

date: 2002/09/30 02:40:52;  author: kei;  state: Exp;
To remove the feature for the original LaserWriter
which causes an annoyance in modrern PS printer, GhostScript, etc.  See
pkg/12904.

--- a2ps.pl-1.45.orig	Fri Mar 15 22:22:10 2002
+++ a2ps.pl-1.45	Fri Mar 15 22:24:14 2002
@@ -496,8 +496,6 @@
 /inch {72 mul} bind def
 ---
     print "%% SelectTray\n";
-    print "statusdict /${paper_tray}tray known { ";
-    print "statusdict begin ${paper_tray}tray end } if\n";
 
     printf("/landscape %s def\n", !$portrait ? "true" : "false");
     printf("/twinpage %s def\n", $twinpage ? "true" : "false");
